ui/jsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+    "compilerOptions": {
+        "baseUrl": "src",
+        "paths": {
+            "@components/*": ["src/components/*"]
+        }
+    }
+}
```

Comment:
This make it so we can use absolute paths to the components directory, and just cleans things up

Reply:
Sounds good to me!